files/generate_ansible_command.py

```diff
@@ -289,6 +302,9 @@ def get_hostname(x):
     else:
         commands_to_run.append('ansible-playbook -D %s' % p)
 
+for l in local_playbooks_to_run:
+    commands_to_run.append('sudo /usr/local/bin/ansible_local.py %s' % l)
```

Comment:
I fail to see what's the goal of the ansible_local.py wrapper instead of running the ansible-playbook command directly here; there's no complex logic in it.
Having the authorized path checked twice means you have to maintain two lists.